massdriver.yaml

```diff
@@ -282,28 +283,33 @@ params:
         enabled:
           type: boolean
           default: false
-          title: "Enabled?"
+          title: Enable
     ttl:
       type: object
-      title: "TTL"
-      description: "Allows you to define a per-item timestamp to determine when an item is no longer needed. Shortly after the date and time of the specified timestamp, DynamoDB deletes the item from your table without consuming any write throughput. This value will be stored as a key called 'TTL'"
+      title: TTL
+      description: Allows you to define a per-item timestamp to determine when an item is no longer needed. Shortly after the date and time of the specified timestamp, DynamoDB deletes the item from your table without consuming any write throughput. This value will be stored as a key called 'TTL'
+      properties:
+        enabled:
+          type: boolean
+          title: Enable
+          default: false
+    pitr:
```

Comment:
You should make this a required field (and regenerate params - notice it is optional).